### PR TITLE
CF-618/CF-619: Highly efficient record processor configuration

### PR DESCRIPTION
**Pull request by mwhahaha**
This pull request includes configuration for the highly efficient record processor filter. https://repose.atlassian.net/wiki/display/REPOSE/Highly+Efficient+Record+Processor+%28HERP%29+filter

### CODE REVIEW

File: templates/highly-efficient-record-processor.cfg.xml.erb [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+<highly-efficient-record-processor xmlns="http://docs.openrepose.org/repose/highly-efficient-record-processor/v1.0"
+                                   pre-filter-logger-name="<%= @pre_filter_logger_name %>"
+                                   post-filter-logger-name="<%= @post_filter_logger_name %>"
+                                   service-code="<%= @service_code %>" region="<%= @region %>" data-center="<%= @datacenter %>">
+    <template crush="<%= @template_crush %>">
+<%- if @template.nil? -%>
+    <!-- todo: methodLabel, host.address -->
+    <![CDATA[
+        <?xml version="1.0"?>
+        <event xmlns:xsi='http://www.w3.org/2001/XMLSchema-instance'
```
Comment (shintasmith) [inline templates/highly-efficient-record-processor.cfg.xml.erb]: This is not standard atom entry, btw. We might want to put down a real atom template as a sample.

Comment (mwhahaha) [inline templates/highly-efficient-record-processor.cfg.xml.erb]: Yea i just ripped this out of the documentation.  If there's something else that needs to be here as a 'default', that should probably be in this.  Alternatively it is overridable.

Comment (shintasmith) [inline templates/highly-efficient-record-processor.cfg.xml.erb]: Here's the full sample of a real atom entry:
https://github.com/rackerlabs/standard-usage-schemas/blob/bdb6e8271c4f9a185f95bfff689b61a2e41336ce/message_samples/corexsd/xml/identity-user-acess-event.xml

Remove line 2.

Comment (mwhahaha) [inline templates/highly-efficient-record-processor.cfg.xml.erb]: OK, I updated the entry.

**Comment by shintasmith** [comment]
lgtm from the template point of view. Do you need Swift to review the puppet stuff? If not, you can just go ahead and merge it

**Pull request merged**